regression/apps/network/udp_err.c

```diff
@@ -198,3 +199,70 @@ FN_TEST(connect)
 	TEST_SUCC(connect(sk_connected, psaddr, addrlen));
 }
 END_TEST()
+
+int set_blocking(int sockfd)
+{
+	int flags = fcntl(sockfd, F_GETFL, 0);
+
+	if (flags == -1) {
+		perror("fcntl get");
+		return -1;
+	}
+
+	if (fcntl(sockfd, F_SETFL, flags & (~O_NONBLOCK)) == -1) {
+		perror("fcntl set");
+		return -1;
+	}
+
+	return 0;
+}
+
+FN_TEST(sendmsg_and_recvmsg)
+{
+	CHECK(set_blocking(sk_connected));
+	CHECK(set_blocking(sk_bound));
```

Comment:
This changes the global state, so I prefer to add a new setup function right before this test:

```c
FN_SETUP(enter_blocking_mode)
{
	set_blocking(sk_connected);
	set_blocking(sk_bound);
}
END_SETUP()
```